assets/component-image-with-text.css

```diff
@@ -367,3 +367,81 @@
     row-gap: var(--grid-desktop-vertical-spacing);
   }
 }
+
+/* Image shapes */
+
+.image-with-text__media.image-shape--circle {
+  clip-path: circle(40%);
+}
+
+.image-with-text__media.image-shape--arch {
+  clip-path: url("#image-shape--arch");
+}
+
+.image-with-text__media.image-shape--parallellogram {
+  clip-path: polygon(15% 0, 100% 0%, 85% 100%, 0% 100%);
+}
+
+.image-with-text__media.image-shape--diamond {
+  clip-path: polygon(50% 0%, 100% 50%, 50% 100%, 0% 50%);
+}
+
+.image-with-text__media.image-shape--rightchevron {
+  clip-path: polygon(85% 0%, 100% 50%, 85% 100%, 0% 100%, 15% 50%, 0% 0%);
+}
+
+.image-with-text__media.image-shape--leftchevron {
+  clip-path: polygon(100% 0%, 85% 50%, 100% 100%, 15% 100%, 0% 50%, 15% 0%);
+}
+
+.image-with-text__media.image-shape--blob {
+  clip-path: polygon(var(--image-shape--blob-5));
+}
+
+/* Image Shapes Hover Rules */
+
+@media (prefers-reduced-motion: no-preference) {
+  .image-with-text__media.image-shape--circle,
+  .image-with-text__media.image-shape--parallellogram,
+  .image-with-text__media.image-shape--diamond,
+  .image-with-text__media.image-shape--leftchevron,
+  .image-with-text__media.image-shape--rightchevron {
+    transition: clip-path var(--duration-default) ease-in-out;
+  }
+
+  .image-with-text__media.image-shape--blob {
+    transition: clip-path var(--duration-long) ease-in-out;
+  }
+
+  .image-with-text__media.image-shape--arch {
+    transition: transform var(--duration-default) ease-in-out;
+  }
+
+  .image-with-text__media.image-shape--circle:hover {
```

Comment:
For non geometric shapes with equal sides, doing a rotation of the path on hover could become clever!

Reply:
Yeah, I felt a little weird about it, and it sounds like you both do too. 😄 I'll remove for now. (94abfcf)

The image here is never clickable, and I don't think it makes sense to tie the animations to scroll in this PR since we'd need to add a JavaScript to handle it. When/if we add other scroll animations (like in #2141) though, we could totally reconsider as a followup PR.